src/documents/management/commands/document_exporter.py

```diff
@@ -292,22 +347,40 @@ def dump(self, progress_bar_disable=False):
                         archive_target,
                     )
 
+            if self.split_manifest:
+                manifest_name = base_name + "-manifest.json"
+                if self.use_filename_prefix:
+                    manifest_name = os.path.join("json", manifest_name)
+                manifest_name = (self.target / Path(manifest_name)).resolve()
+                manifest_name.parent.mkdir(parents=True, exist_ok=True)
+                content = [document_manifest[index]]
+                content += list(
+                    filter(
+                        lambda d: d["fields"]["document"] == document_dict["pk"],
+                        comments,
+                    ),
+                )
+                manifest_name.write_text(json.dumps(content, indent=2))
+                if manifest_name in self.files_in_export_dir:
+                    self.files_in_export_dir.remove(manifest_name)
+
         # 4.1 write manifest to target folder
         manifest_path = (self.target / Path("manifest.json")).resolve()
         manifest_path.write_text(json.dumps(manifest, indent=2))
```

Comment:
When I set `--split-manifest` the original `manifest.json` is also created still.  Probably needs a check around here?

Reply:
Yes, this is expected. Information about documents are in dedicated json files, but application settings are still in the main manifest (tags, correspondent, documenttype, etc)

Reply:
Ah, I see.  Thanks

Reply:
But maybe, if you wondered why, others will too. And then we should explain that in the documentation

Reply:
see 16f008d5c6d12b0114a2f07967d7d8e7eeb3d52b